File: JavaScript projects/JavaScript Basics/Conditional Statements/exercises/08_lunch_break.js
function lunchBreak(seriesName, timeEpisode, timeBreak){
    let timeLunch = timeBreak * 0.125;
    let timeRecovery = timeBreak * 0.25;
    let neededTime = timeEpisode + timeLunch + timeRecovery;

    if (neededTime <= timeBreak){
        console.log(`You have enough time to watch ${seriesName} and left with ${Math.ceil(timeBreak - neededTime)} minutes free time.`);
    }else{
        console.log(`You don't have enough time to watch ${seriesName}, you need ${Math.ceil(neededTime - timeBreak)} more minutes.`);
    }
}

lunchBreak("Teen Wolf", 48, 60);

File: Python projects/Python Basics/02_conditional_statements/exercises/08_lunch_break.py
import math

name = str(input())
time_episode = int(input())
time_break = int(input())
time_lunch = time_break * 1 / 8
time_recover = time_break * 1 / 4
time_needed = time_episode + time_lunch + time_recover

if time_break >= time_needed:
    print(
        f"You have enough time to watch {name} and left with {math.ceil(time_break - time_needed)} minutes free time.")
else:
    print(f"You don't have enough time to watch {name}, you need {math.ceil(time_needed - time_break)} more minutes.")
